camera_error_handler.py

```diff
@@ -0,0 +1,34 @@
+"""
+Exception handler for the Raspicam 
+
+Used alongside the 'with' magic Python word.
+eg. with CameraException(self.client, self.backend_name)
+"""
+from json import dumps
+from traceback import format_exc
+
+from config import read_configs
+from topics import Camera
+
+class CameraException:
```

Comment:
While I like your thinking, I think the name `CameraException` is misleading. All the classes I can think of that end in `Exception` (or `Error`, btw) are ones which can be raised. (This is the definition of an exception.) However, `CameraException` is not an exception that can be raised - rather it is an exception handler, because rather than being an error itself, it takes some error that has occured and deals with it. I reckon `CameraErrorHandler` would be a good name!

tl;dr we can't `raise CameraException` so exception it is not.